packages/flutter_tools/templates/cocoapods/Podfile-swift

```diff
@@ -22,7 +24,6 @@ def parse_KV_file(file,seperator='=')
 end
 
 target 'Runner' do
-  use_frameworks!
```

Comment:
As far as I understand, we still need to keep this for swift

Reply:
Indeed, for Swift code it is mandatory to use dynamic frameworks instead of static libraries. http://blog.cocoapods.org/CocoaPods-0.36/